### ISSUE TITLE
What should be included in the extension?

### ISSUE THREAD

**Issue by cwhite911** [tomorrownow/grass-stac-extension#1]
Let's use this comment board to discuss what additional metadata should be included in the grassdata STAC.

**Comment by m-mohr** [comment]
Quite a number of fields have general definitions in other more commonly used extensions that should be re-used so that the interoperability really works and clients can make use of them.

grass:description -> description
grass:creator -> providers
grass:cols -> proj:shape
grass:min/max -> raster:bands.min?
grass:color_table/ncats -> classification:classes?
...

**Comment by cwhite911** [comment]
@m-mohr I agree that we should make sure to map to as many other commonly used extensions as possible. Once we agree on a specification we create open a t.out.stac feature to export a either a grassdata base, location, or mapset.
